stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/adapters/SimpleListAdapter.kt

```diff
@@ -1,34 +1,34 @@
-package io.getstream.chat.android.ui.common.internal
+package io.getstream.chat.android.ui.common.adapters
 
 import android.content.Context
 import android.view.View
 import androidx.annotation.CallSuper
 import androidx.recyclerview.widget.RecyclerView
 
-internal abstract class SimpleListAdapter<T : Any, VH : SimpleListAdapter.ViewHolder<T>> : RecyclerView.Adapter<VH>() {
-    var itemList: MutableList<T> = mutableListOf()
+public abstract class SimpleListAdapter<T : Any, VH : SimpleListAdapter.ViewHolder<T>> : RecyclerView.Adapter<VH>() {
```

Comment:
I'd prefer not exposing `SimpleListAdapter` (imagine that people start using this as the base adapter for random pieces of UI unrelated to our SDK, because they don't realize where they're importing it from).

If I understand it correctly, we could avoid this by creating something like an `AttachmentViewHolder` class that has `bind/unbind` methods.